File: easy/position.rb
File.open(ARGV[0]).each_line do |line|
  s = line.split(',').map(&:to_i)
  puts s[0] & 1 << s[1] - 1 > 0 == s[0] & 1 << s[2] - 1 > 0
end

File: easy/position.pl
use strict;
open( INFILE, "<", $ARGV[0] )
  or die("Cannot open file $ARGV[0] for reading: $!");
while ( my $line = <INFILE> ) {
    my ( $x, $p, $q ) = ( $line =~ /(\d+),(\d+),(\d+)/ );
    print( ( $x & 1 << $p - 1 ) > 0 == ( $x & 1 << $q - 1 ) > 0
        ? "true\n"
        : "false\n" );
}
close(INFILE);
